src/implementations/vanilla/components/file/file.scss

```diff
@@ -259,10 +259,6 @@ $file: null !default;
     width: auto;
   }
 
-  .ecl-file__info {
```

Comment:
is this a needed fix? In that case i would also apply this as a regular fix on v4-dev directly without depending on this pr